chia/rpc/wallet_rpc_client.py

```diff
@@ -958,8 +958,8 @@ async def count_nfts(self, wallet_id: Optional[int]):
         response = await self.fetch("nft_count_nfts", request)
         return response
 
-    async def list_nfts(self, wallet_id):
-        request: Dict[str, Any] = {"wallet_id": wallet_id, "num": 100_000}
+    async def list_nfts(self, wallet_id, num=100_000, start_index=0):
```

Comment:
100k limit was there because we didn't have pagination, it's better to move this back to `50`. 
```suggestion
async def list_nfts(self, wallet_id, num=50, start_index=0):
```